pippo-session-parent/pippo-session-hazelcast/src/test/java/ro/pippo/session/hazelcast/HazelcastSessionDataStorageTest.java

```diff
@@ -65,18 +65,19 @@ public void testCreate() {
      */
     @Test
     public void testSave() {
-        HazelcastInstance hazelcastInstance2 = Hazelcast.newHazelcastInstance();
+//        HazelcastInstance hazelcastInstance2 = Hazelcast.newHazelcastInstance();
```

Comment:
@barrantesgerman I remarked that if I don't comment the second instance of Hazelcast, I obtain a test fail (`saved` is null). I don't understand why?